tests/unittests/PartitionerTest.cpp

```diff
@@ -435,13 +475,13 @@ TEST_F(PartitionerTest, Basic1Roofline) {
       {"fc_dot4", 512},
       {"fc_add_bias4", 32},
   };
-  for (auto const &p : nodeNamesMap) {
-    auto *N = p.first;
-    EXPECT_EQ(myPartitioner.getMemUsage(N), expectedMemUsage[p.second]);
-  }
+  // for (auto const &p : nodeNamesMap) {
+  //   auto *N = p.first;
+  //   EXPECT_EQ(myPartitioner.getMemUsage(N), expectedMemUsage[p.second]);
+  // }
```

Comment:
Same. I think the checking should be here.

Reply:
fixed too.